File: Entidades/InformacionPersonal.vb
'Continene: Id_Persona, Direccion, Observaciones
Public Class InformacionPersonal

    'Se declara un constructor para inicializar la lista de teléfonos de una persona
    Public Sub New()

    End Sub

    'Variable extraída de la tabla PERSONA y que específica a cual persona pertenece la información.
    Private _varIdPersona As Integer
    Public Property IdPersona() As Integer
        Get
            Return _varIdPersona
        End Get
        Set(ByVal value As Integer)
            _varIdPersona = value
        End Set
    End Property

    'La direccion exacta donde se puede localizar al cliente.
    Private _varDireccion As String
    Public Property Direccion() As String
        Get
            Return _varDireccion
        End Get
        Set(ByVal value As String)
            _varDireccion = value
        End Set
    End Property

    'Alguna otra cosa que se debe saber del cliente.
    Private _varObservaciones As String
    Public Property Observaciones() As String
        Get
            Return _varObservaciones
        End Get
        Set(ByVal value As String)
            _varObservaciones = value
        End Set
    End Property

    Private _varTelefonoPrincipal As String
    Public Property TelefonoPrincipal() As String
        Get
            Return _varTelefonoPrincipal
        End Get
        Set(ByVal value As String)
            _varTelefonoPrincipal = value
        End Set
    End Property

    Private _varOtroTelefono As String
    Public Property OtroTelefono() As String
        Get
            Return _varOtroTelefono
        End Get
        Set(ByVal value As String)
            _varOtroTelefono = value
        End Set
    End Property

    Private _varFaxPrincipal As String
    Public Property FaxPrincipal() As String
        Get
            Return _varFaxPrincipal
        End Get
        Set(ByVal value As String)
            _varFaxPrincipal = value
        End Set
    End Property

    Private _varEmail As String
    Public Property Email() As String
        Get
            Return _varEmail
        End Get
        Set(ByVal value As String)
            _varEmail = value
        End Set
    End Property

    'Almacena la dirección del sitio web del cliente.
    Private _varSitioWeb As String
    Public Property SitioWeb() As String
        Get
            Return _varSitioWeb
        End Get
        Set(ByVal value As String)
            _varSitioWeb = value
        End Set
    End Property
End Class

File: Entities/InformacionPersonal.cs
namespace Entidades
{
    using System;

    public class InformacionPersonal
    {
        private int _varIdPersona;
        private string _varDireccion;
        private string _varObservaciones;
        private string _varTelefonoPrincipal;
        private string _varOtroTelefono;
        private string _varFaxPrincipal;
        private string _varEmail;
        private string _varSitioWeb;

        public int IdPersona
        {
            get => 
                this._varIdPersona;
            set => 
                this._varIdPersona = value;
        }

        public string Direccion
        {
            get => 
                this._varDireccion;
            set => 
                this._varDireccion = value;
        }

        public string Observaciones
        {
            get => 
                this._varObservaciones;
            set => 
                this._varObservaciones = value;
        }

        public string TelefonoPrincipal
        {
            get => 
                this._varTelefonoPrincipal;
            set => 
                this._varTelefonoPrincipal = value;
        }

        public string OtroTelefono
        {
            get => 
                this._varOtroTelefono;
            set => 
                this._varOtroTelefono = value;
        }

        public string FaxPrincipal
        {
            get => 
                this._varFaxPrincipal;
            set => 
                this._varFaxPrincipal = value;
        }

        public string Email
        {
            get => 
                this._varEmail;
            set => 
                this._varEmail = value;
        }

        public string SitioWeb
        {
            get => 
                this._varSitioWeb;
            set => 
                this._varSitioWeb = value;
        }
    }
}

